Readme.md

```diff
@@ -38,9 +42,20 @@ Additional Configuration values used:
 
     # alternatively: set a fixed bucket, no need for bucket name in Image-Path
     S3_LOADER_BUCKET = 'thumbor-images'
+    # A root path for loading images, useful if you share the bucket
+    S3_LOADER_ROOT_PATH = 'source-images'
 
     # configuration settings specific for the storages
 
+    STORAGE_BUCKET = 'thumbor-images'
```

Comment:
It would be nice at some point to move all config values to `AWS_` or `S3_` prefix to prevent collisions.